apps/ae_socket_connector/test/aesocketconnector_test.exs

```diff
@@ -1,6 +1,7 @@
 defmodule SocketConnectorTest do
```

Comment:
This file should use the underscores in the name too.

Reply:
fixed